tests/basetest.rb

```diff
@@ -137,7 +137,23 @@ def teardown
     @device = nil
   end
 
+  # Execute the specified config commands and warn if the
+  # ouput matches the following regex: /invalid|^%/i
```

Comment:
s/ouput/output

Also probably not worth including the actual regex in the comment in addition to below - seems likely to become out of sync over time.